### PR TITLE
Add map manual refresh parameter

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -83,6 +83,8 @@
     "Limit": "Limit",
     "load": "load",
     "Load": "Load",
+    "MapManualRefresh": "Manual update of geographical view\u00a0:",
```
Comment (EstherDarkish) [inline src/translations/en.json:86]: There is no space before colons in english.

**Review by EstherDarkish — APPROVED** [review]

File: src/translations/en.json (hunk 1 above)
Comment (Meklo) [reply src/translations/en.json]: ✓

File: src/components/study-container.js [hunk 2]
```diff
@@ -542,44 +556,57 @@ export function StudyContainer({ view, onChangeTab }) {
...
-                const substationsIds =
-                    studyUpdatedForce.eventData.headers['substationsIds'];
-                const tmp = substationsIds.substring(
-                    1,
-                    substationsIds.length - 1
-                ); // removing square brackets
-                if (tmp && tmp.length > 0) {
-                    updateNetwork(tmp.split(', '));
-                }
+        if (!mapManualRefresh || !reloadMapNeeded) {
```
Comment (niconoir) [inline src/components/study-container.js:559]: I am not sure if the modifications made in this file are relevant. If the network is not reloaded when currentNode changes or when network has been updated on the back side, the data displayed in the equipment table might be wrong.

**Review by niconoir — CHANGES_REQUESTED** [review]

File: src/components/study-container.js (hunk 2 above)
Comment (Meklo) [reply src/components/study-container.js]: I implemented a fix which reload network data when the user switch to the spreadsheet
If the user is in manual mode, the spreadsheet is already open, and new data comes in therefore the data is invalidated

File: src/components/study-container.js [hunk 3]
```diff
@@ -352,6 +364,18 @@ export function StudyContainer({ view, onChangeTab }) {
         // Note: studyUuid and dispatch don't change
     }, [studyUuid, dispatch, snackError, snackWarning]);
 
+    useEffect(() => {
```
[the remark this answers is not in the document]
Comment (Meklo) [reply src/components/study-container.js:367]: One useEffect is there to handle manual mode triggers and the other is there to handle automatic mode triggers
In practice I have verified that during network and equipment manipulations there are no overlap between the two, the network reload only once when needed.
It's on the app startup that there might be multiple call to network loading.
I'm currently looking for a fix.

[the remark this answers is not in the document]
Comment (Meklo) [reply src/components/study-container.js:367]: It's fixed

**Review by niconoir — APPROVED** [review]

**Pull request merged**